packages/iov-core/src/jsonrpcsigningserver.ts

```diff
@@ -156,10 +156,12 @@ export class JsonRpcSigningServer {
           };
           break;
         case "signAndPost":
+          // Simplify when https://github.com/iov-one/iov-core/issues/929 is done
```

Comment:
Why not do this directly?

Reply:
Because #929 is a breaking change of the API (caller code like `result === undefined` would break). In this case nobody would care or notice but I am a bit semver-obsessed.